### PR TITLE
(Pol) Start implementing smart paradigms for nouns

### PR DESCRIPTION
**Pull request by inariksit**

### CODE REVIEW

**Comment by ppski** [comment]
This looks nice! Will this be implemented eventually?

**Comment by inariksit** [comment]
It's not a priority at the moment. But given that I started writing [slides about it](https://docs.google.com/presentation/d/1WDTAUJjIvFAdfgdON4pnXb3aBjuKKtPcqM5FxF3qbjw/edit?usp=sharing) for the GF summer school (back when I thought it would be feasible to travel to Singapore in 2020 :-D), there's hope. Just by previous experience how my motivation works, unless there's some external pressure, I'd say I will get motivated about the project around 2 days before it's time to give that presentation.

**Comment by inariksit** [comment]
Thanks for the patch @ppski ! Do you want me to merge this to master?

**Comment by ppski** [comment]
@inariksit Yes, please! My patch doesn't add too much, but I think at least these basic smart paradigms are an improvement over the more manual tables.

File: src/polish/ParadigmsPol.gf [hunk 1]
```diff
@@ -1,28 +1,386 @@
...
+        <_,_ + "i"> => NM.mkNTable0995 sgnom ; -- Alternatives: mkNTable0995,mkNTable0583
+        <_,_ + "u"> => NM.mkNTable0171 sgnom ; -- Alternatives: mkNTable0696,mkNTable0111,mkNTable0539,mkNTable0247
+        <_,_ + "y"> => NM.mkNTable0550 sgnom ;
+        <_,_ + "ia"> => NM.mkNTable0803 sgnom  ;-- Alternative: mkNTable0648
+        <_,_ + "iu"> => NM.mkNTable0662 sgnom ;
+        <_,_ + "na"> => NM.mkNTable0286 sgnom ;
+        <_,_ + "ego"> => NM.mkNTable0589 sgnom ; -- Alternative: mkNTable0966
+        <_,_> => guess_paradigm_basic sgnom}; 
+     -- 1 string
+     guess_paradigm : (sgnom : Str) -> SubstForm => Str = guess_paradigm_basic
```
Comment (ppski) [inline src/polish/ParadigmsPol.gf:148]: There's a mistake here! Will push a fix.

Comment (inariksit) [reply src/polish/ParadigmsPol.gf:148]: Sorry, it's not a mistake, just [eta-reduction](https://wiki.haskell.org/Eta_conversion). But I understand if it confuses people, so feel free to fix it to 

```haskell
guess_paradigm : (sgnom : Str) -> SubstForm => Str = \sgnom -> guess_paradigm_basic sgnom
```

**Pull request merged**